src/codewhisperer/util/runtimeLanguageContext.ts

```diff
@@ -65,6 +65,7 @@ export class RuntimeLanguageContext {
             javascript: 'javascript',
             javascriptreact: 'jsx',
             json: 'json',
+            jsonc: 'jsonc',
```

Comment:
These were grabbed from the VSCode documentation on `.jsonc` [here](https://github.com/microsoft/vscode/blob/42b9e864678e6d0250edaa7c25c17e1cdf3d04bf/extensions/json-language-features/server/README.md?plain=1#L13)